src/ModalHeader.js

```diff
@@ -59,14 +59,14 @@ class ModalHeader extends React.Component {
 
     return (
       <div {...elementProps} className={classNames(className, classes)}>
+        {children}
+
         {closeButton && (
           <CloseButton
             label={closeLabel}
             onClick={createChainedFunction(modal && modal.onHide, onHide)}
```

Comment:
We want to update the `context.$bs_modal` usage to use the new `React.createContext` API outlined here: https://github.com/acdlite/rfcs/blob/new-version-of-context/text/0000-new-version-of-context.md

Reply:
Is there an example I can look at in another component?